src/Components/Home.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+function Home() {
+  return (
+    <main>
+      <h2>Welcome to our page!</h2>
+      <p className="welcome">
+        {' '}
+        lorem Udemy is a global marketplace for online courses of all shapes and sizes.
+        From data science to graphic design and social media marketing, there are thousands
+        of programs available for learning and teaching online. By enabling people to study
+        from the comfort of their own homes, Udemy strives to help individuals enhance their
+        skills and realize their visions. While users are able to individually sign up for courses,
+        this particular landing page promotes Udemy’s business plan. It offers companies a platform
+        which they can train and educate their employees, providing them with extra capabilities and
+        on a range of topics. This can help businesses support their employees while working.
+      </p>
+    </main>
+  );
+}
+
```

Comment:
[REQUIRED] Kindly align the texts the left just like the wireframe

Reply:
I have implemented the changes